static_src/stores/service_instance_store.js

```diff
@@ -37,6 +37,15 @@ class ServiceInstanceStore extends BaseStore {
         break;
       }
 
+      case serviceActionTypes.SERVICE_INSTANCE_RECEIVED: {
+        const instance = this.formatSplitResponse(
+          [action.serviceInstance])[0];
```

Comment:
question (it's been awhile since i've reviewed some of this code so i could be off or forgot something)
below we use `fetchServiceInstance` which uses `fetchOne`. Why do we need to still treat it as an array and return the first one via `[0]`.